deploy/mock-identity-system/install.sh

```
@@ -59,14 +59,88 @@ function installing_mock-identity-system() {
+      echo "Invalid input. Please enter 'y' or 'n'."
+    fi
+  done
+
+  HSM_HELM_ARGS=''
+  if [[ $enable_volume == 'true' ]]; then
+    default_volume_size=100M
+    read -p "Provide the size for volume [ default : 100M ]" volume_size
+    volume_size=${volume_size:-$default_volume_size}
+
+    default_volume_mount_path='/home/mosip/config/'
+    read -p "Provide the mount path for volume [ default : '/home/mosip/config/' ] : " volume_mount_path
```

Shellcheck (0.11.0)

[info] read without -r will mangle backslashes.

(SC2162)